### PR TITLE
Upgraded to ES6

### PR DESCRIPTION
**Pull request by rek**
This fork is based off the work EddM did.

I tried not to change much, but there are some slight filter changes in the example etc.

Upgraded webpack and added lint rules too.

### CODE REVIEW

**Review by michaelbrewerdavis — COMMENTED** [review]
Can you describe the errant behavior here?  The change makes sense but I'd like to verify the change.

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
+{
```
Comment (michaelbrewerdavis) [inline .eslintrc.json]: Not sure if other contributors agree, but for this project I'd prefer to use a packaged set of linter rules to an ad hoc list.

File: .babelrc [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-		"development": {
-			"plugins": [
-				"react-hot-loader/babel"
-			]
-		}
-	}
+    "presets": [
+        ["env", {
+            "targets": {
+                "browsers": ["last 2 versions", "safari >= 7"]
```
Comment (michaelbrewerdavis) [inline .babelrc]: I'm out of my depth in reviewing the build system changes, but this feels like a breaking change?  What is the motivation for adding a restriction to the client app?

**Comment by rek** [comment]
Sorry, forgot to mention, I make the changes originally because the react version was out of date and throwing warnings in 15, thus it would not work in 16.

File: .eslintrc.json (hunk 1 above)
Comment (rek) [reply .eslintrc.json]: No problem, I just used what I had lying around. A packaged set sounds great.

**Comment by ryankshaw** [comment]
with the 2.5.0 release we just made, this can be closed now

**Pull request closed without merging**